parcelgen-runtime/src/com/yelp/parcelgen/JsonUtil.java

```diff
@@ -147,16 +147,16 @@ public static Date parseTimestamp(JSONObject object, String key) {
     }
 
     /**
-     * @return An ArrayList of the elements of array converted to strings, or the empty list if
+     * @return An mutable ArrayList of the elements of array converted to strings, or the empty list if
```

Comment:
Note:
If someone is using this library and making following comparison: 
Collections.emptyList() == getStringList(array), then it will break lot of things provided array is empty